### ISSUE TITLE
moving to new AWS Zone

### ISSUE THREAD

**Issue by gauteaasolaas** [Esri/mdcs-py#64]
We have all our mrf-data in eu-west-1 but are currently moving it to eu-north-1 to get lower latency in Norway.
This means that all our mosaics with embedded proxies need to change the bucket reference inside the embedded proxies.
Currently the real path to the mrf-data is something like this: euwestbucket/subfolder/mrf/...
In the future it will look something like this: eunorthbucket/subfolder/mrf/...
I've tried to export paths with mdtools, but only the old external mrfproxy-paths are shown e.g.:
/servicedata/Markfuktighet_DTW/mrfproxy/Markfuktighet_DTW.Overviews/Ov_i04_L01_R000000AE_C0000003B.tif
So my guess is that I need to add some special formatting before the bucketname to allow the MD to know where to look.
But what formatting is that?

**Comment by gauteaasolaas** [comment]
I've tried this out on a small project:
D:\work\mrf>"C:\Program Files (x86)\ArcGIS\Desktop10.7\bin\mdtools.exe" --md="D:\work\mrf\MosaicDataset_ep.gdb\s_1397_ep" --replace_paths --oldpath="gd-data.s3-eu-west-1." --newpath="geodata-gdonline-gdo-processing-ortho.s3-eu-north-1."
. Initializing ArcObjects...
. Loading mosaic dataset: D:\work\mrf\MosaicDataset_ep.gdb\s_1397_ep
. Replacing paths in moasic dataset items...
... Old Path: gd-data.s3-eu-west-1.
... New Path: geodata-gdonline-gdo-processing-ortho.s3-eu-north-1.
. Processing Complete. Selected: 8 items, updated: 8 items.
. Application executed in 5.14 s.
. Shutting down ArcObjects.
D:\work\mrf>
Encouraging even if there are 13 rasters in the project, not 8 since the original rasters have been merged.
However, when I removed the mrf-folder form the old bucket, the msoaic dataset lost access  to the data.
The buckets are configured with access through the same credentials.
